mcmillan_airfield/env-hooks/99_mcmillan_airfield_setup.sh.in

```diff
@@ -0,0 +1,3 @@
+ament_prepend_unique_value GAZEBO_MODEL_PATH "@CMAKE_INSTALL_PREFIX@/share/@PROJECT_NAME@/models"
```

Comment:
This is the only instance of a setup file being added back. Is it unneeded here or needed everywhere else?

Reply:
I forgot to remove it